<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createSavingsAccount</class>
 <widget class="QWidget" name="createSavingsAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>createSavingsAccount</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="createAccountButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>create savings account</string>
   </property>
  </widget>
  <widget class="QLabel" name="instructionlabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Please fill the following form :</string>
   </property>
  </widget>
  <widget class="QLabel" name="heading">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create Savings Account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountNoEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Occupation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="accountlabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Name (as per CNIC)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>420</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Date of Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Phone number</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultlabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Country</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Email Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>CNIC</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-135</x>
     <y>-129</y>
     <width>1021</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/guidesign/BackgroundImage.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>createAccountButton</zorder>
  <zorder>instructionlabel</zorder>
  <zorder>heading</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>accountNoEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>lineEdit</zorder>
  <zorder>dateEdit</zorder>
  <zorder>accountlabel</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>resultlabel</zorder>
  <zorder>label_2</zorder>
  <zorder>label_7</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bankingsystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>createAccountButton</sender>
   <signal>clicked()</signal>
   <receiver>createSavingsAccount</receiver>
   <slot>onCreateSavingsAccountClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>createSavingsAccount</receiver>
   <slot>goBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCreateSavingsAccountClicked()</slot>
  <slot>goBack()</slot>
 </slots>
</ui>
